Corona-Warn-App/src/main/java/de/rki/coronawarnapp/familytest/core/repository/BaseCoronaTestProcessor.kt

```diff
@@ -115,20 +108,10 @@ class BaseCoronaTestProcessor @Inject constructor(
             type = VerificationKeyType.GUID,
         )
     }
-}
 
-private fun CoronaTest.isPollingStopped(forceUpdate: Boolean, now: Instant): Boolean =
-    (!forceUpdate && testResult in finalStates) || isOlderThan21Days(now) && testResult in redeemedStates
-
-private val finalStates = setOf(
-    PCR_POSITIVE,
-    PCR_NEGATIVE,
-    PCR_OR_RAT_REDEEMED,
-    RAT_REDEEMED,
-    RAT_POSITIVE,
-    RAT_NEGATIVE,
-    PCR_INVALID,
-    RAT_INVALID
-)
-
-private val redeemedStates = setOf(PCR_OR_RAT_REDEEMED, RAT_REDEEMED)
+    data class CoronaTestResultUpdate(
+        val coronaTestResult: CoronaTestResult,
+        val sampleCollectedAt: Instant? = null,
+        val labId: String? = null,
+    )
```

Comment:
Do we need to show it? Thomas said that we do show an error instead of the cached test result is a problem in Android, iOS does it differently.

Reply:
In the list we have `swipe to refresh`, when user triggers the refresh, we should display an error dialog or so in case the process did not go through. otherwise user will think nothing changed. for auto refresh we can just ignore them